openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -0,0 +1,52 @@
+"""
+This file contains utility functions which will responsible for sending emails.
+"""
+import logging
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.mail import send_mail
+from django.utils.translation import ugettext as _
+
+from edxmako.shortcuts import render_to_string
+from microsite_configuration import microsite
+from openedx.core.djangoapps.user_api.accounts.api import get_account_settings
+from xmodule.modulestore.django import modulestore
+
+
+log = logging.getLogger(__name__)
+
+
+def send_credit_notifications(username, course_key, email_type="credit_eligibility"):
+    """Sends email notification to user on different phases during credit
+    course e.g. credit eligibility, credit payment etc.
+    """
+    user = None
+    try:
+        user = User.objects.get(username=username)
+    except User.DoesNotExist:
+        log.error('No user with %s exist', username)
+        return
+
+    account_settings = get_account_settings(user)
+    course = modulestore().get_course(course_key, depth=0)
+    course_display_name = course.display_name
+    # TODO: change context according to email type
+    context = {
+        'full_name': account_settings['name'],
+        'platform_name': settings.PLATFORM_NAME,
+        'course_name': course_display_name,
+    }
+
+    from_address = microsite.get_value('default_from_email', settings.DEFAULT_FROM_EMAIL)
+    to_address = account_settings['email']
+
+    # TODO: add/change message file (path), subject and message according to email type
+    if email_type == "credit_receipt":
+        subject = _("Credit Course Payment Receipt")
+        message = render_to_string('emails/credit_payment_receipt.txt', context)
+    else:
+        subject = _("Course Credit Eligibility")
+        message = render_to_string('emails/credit_eligibility_email.txt', context)
+
+    send_mail(subject, message, from_address, [to_address], fail_silently=False)
```

Comment:
You don't necessarily need to do it in this PR, but please keep in mind that the acceptance criteria for these stories include:

* Including both HTML and plaintext versions of the email.
* Adding a tracking pixel to the HTML version (so we can track when the user opens the email).